pkg/auth/credentials.go

```diff
@@ -64,13 +67,29 @@ func isSecretKeyValid(secretKey string) bool {
 
 // Credentials holds access and secret keys.
 type Credentials struct {
-	AccessKey string `json:"accessKey,omitempty"`
-	SecretKey string `json:"secretKey,omitempty"`
+	AccessKey    string    `xml:"AccessKeyId" json:"accessKey,omitempty"`
+	SecretKey    string    `xml:"SecretAccessKey" json:"secretKey,omitempty"`
+	Expiration   time.Time `xml:"Expiration" json:"expiration,omitempty"`
+	SessionToken string    `xml:"SessionToken" json:"sessionToken,omitempty"`
```

Comment:
After adding SessionToken, shouldn't we also update Credentials.Equal() function ?

Reply:
We can but it's not used anywhere might as well remove it.